omod/src/main/java/org/openmrs/module/pihcore/page/controller/visit/TodaysVisitsPageController.java

```diff
@@ -5,25 +5,28 @@
 import org.openmrs.module.emrapi.domainwrapper.DomainWrapperFactory;
 import org.openmrs.ui.framework.annotation.SpringBean;
 import org.openmrs.ui.framework.page.PageModel;
+import org.openmrs.module.appui.UiSessionContext;
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.util.Date;
 import java.util.List;
+import java.util.Collections;
 
 public class TodaysVisitsPageController {
 
     public void get(PageModel model,
                     @SpringBean("visitService") VisitService visitService,
-                    @SpringBean("domainWrapperFactory")DomainWrapperFactory domainWrapperFactory) {
+                    @SpringBean("domainWrapperFactory")DomainWrapperFactory domainWrapperFactory,
+                    UiSessionContext uiSessionContext) {
 
         LocalDate todayDate = LocalDate.now();
         Date startOfDayToday = java.sql.Timestamp.valueOf(LocalDateTime.of(todayDate, LocalTime.MIN));
         Date endOfDayToday = java.sql.Timestamp.valueOf(LocalDateTime.of(todayDate, LocalTime.MAX));
 
         // all non-voided visits that have been started any time today
```

Comment:
let's update this comment to note that we are now limiting by session location so we don't get confused in the future, ie something like:

// all non-voided visits that have been started any time today at the current session location.